parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -293,7 +320,8 @@ pub mod pallet {
 
 			let ticket =
 				OutboundQueueTicket { id: message_id, origin: message.origin, message: encoded };
-			Ok((ticket, T::Fee::get()))
+
+			Ok((ticket, delivery_fee))
```

Comment:
should it be `T::Fee::get() + delivery_fee`?  I would assume the `T::Fee::get()` to cover the cost in bridgeHub(previously I named as `base_fee` not only for the cost of submit message but also the cost for constructing the merkle tree when messages are committed).

Reply:
It just covers the cost on the Ethereum side for now. Can always do a follow up PR to improve, but its not necessary for the audit.